graylog2-server/src/main/java/org/graylog2/rest/resources/users/UsersResource.java

```diff
@@ -423,11 +421,14 @@ public void changePassword(
 
     @GET
     @Path("{username}/tokens")
-    @RequiresPermissions(RestPermissions.USERS_TOKENLIST)
     @ApiOperation("Retrieves the list of access tokens for a user")
     public TokenList listTokens(@ApiParam(name = "username", required = true)
                                 @PathParam("username") String username) {
-        final User user = _tokensCheckAndLoadUser(username);
+        final User user = _tokensLoadUser(username);
```

Comment:
I think the naming of that method is suboptimal, for two reasons:

  * Method names are not prefixed with `_` in Java. The signature already reveals that it is a private method.
  * The name `_tokensLoadUser` implies that it is specific to the Token management, which it is not. If it would be called `loadUser`, it would describe accurately what it does and it could be reused by the `changeUser` method.